Update fr_fr.json

XtraBe opened the pull request.

XtraBe committed:
Update fr_fr.json

misieur commented on common/src/main/resources/assets/iris/lang/fr_fr.json:
```diff
@@ -1,26 +1,75 @@
+  "options.iris.reset.tooltip.holdShift": "Maintenez SHIFT et cliquez pour réinitialiser",
+  "options.iris.importSettings.tooltip": "Importer les paramètres depuis un fichier",
+  "options.iris.exportSettings.tooltip": "Exporter les paramètres appliqués vers un fichier",
+  "options.iris.mustDisableFullscreen": "Veuillez d'abord désactiver le plein écran!",
```
"options.iris.mustDisableFullscreen": "Veuillez d'abord désactiver le plein écran !",

misieur commented on common/src/main/resources/assets/iris/lang/fr_fr.json:
```diff
@@ -1,26 +1,75 @@
+  "options.iris.shaders.nonePresent": "Shaders : Aucun pack présent",
+  "options.iris.back": "Retour",
+  "options.iris.reset": "Réinitialiser",
+  "options.iris.reset.tooltip": "Réinitialiser TOUTES les options et appliquer?",
```
"options.iris.reset.tooltip": "Réinitialiser TOUTES les options et appliquer ?",

XtraBe replied:
Vous souhaiteriez indiquer : "Basculer TOUTES les options et appliquer ?" ? 
Je ne comprend pas votre remarques avec Basculer. et la solution que vous proposer est la meme traduction. ;)

misieur replied:
Non il faut juste un espace avant le point d’interrogation 
"appliquer?" -> "appliquer ?"

misieur commented on common/src/main/resources/assets/iris/lang/fr_fr.json:
```diff
@@ -1,26 +1,75 @@
+  "options.iris.importSettings.tooltip": "Importer les paramètres depuis un fichier",
+  "options.iris.exportSettings.tooltip": "Exporter les paramètres appliqués vers un fichier",
+  "options.iris.mustDisableFullscreen": "Veuillez d'abord désactiver le plein écran!",
+  "options.iris.setToDefault": "Réinitialiser l'option à sa valeur par défaut?",
```
"options.iris.setToDefault": "Réinitialiser l'option à sa valeur par défaut ?",

misieur commented on common/src/main/resources/assets/iris/lang/fr_fr.json:
```diff
@@ -1,26 +1,75 @@
+  "options.iris.shadowDistance": "Distance maximale des ombres",
+  "options.iris.shadowDistance.enabled": "Vous permet de changer la distance maximale pour les ombres. Le terrain et les entités au-delà de cette distance ne projettent pas d'ombre. Réduire la distance peut améliorer significativement les performances.",
+  "options.iris.shadowDistance.disabled": "Votre pack de shaders actuel a déjà défini une distance de rendu pour les ombres; vous ne pouvez pas la changer.",
+  "options.iris.shadowDistance.sodium_tooltip": "La distance de rendu des ombres contrôle jusqu'où le terrain peut être rendu dans le passage des ombres. Des distances plus faibles signifient que moins de terrain sera rendu, améliorant les performances. Cette option ne peut pas être modifiée sur les packs qui définissent explicitement une distance. La distance réelle est plafonnée par le paramètr ...
```
"options.iris.shadowDistance.sodium_tooltip": "La distance de rendu des ombres contrôle jusqu'où le terrain peut être rendu dans le passage des ombres. Des distances plus faibles signifient que moins de terrain sera rendu, améliorant les performances. Cette option ne peut pas être modifiée sur les packs de shader qui définissent explicitement une distance. La distance réelle est plafonnée par le paramètre de distance de vue.",

misieur reviewed (commented):
The translations are pretty good, but there are some small mistakes and some words like `basculer` are not very suitable, but these are the only correct translations.

XtraBe committed:
Update of the different spaces in FR punctuation.

Update of the Non-breaking Space:
• "name": "Non-breaking space",
• "unicode": "U+00A0",
• "character": " ",
• "json": "\u00A0"

and Narrow Non-breaking Space:
• "name": "Narrow no-break space",
• "unicode": "U+202F",
• "character": " ",
• "json": "\u202F"
